packages/web/src/components/edit-folder-modal/EditFolderModal.tsx

```diff
@@ -12,13 +12,13 @@ import { useDispatch } from 'react-redux'
 
 import { useModalState } from 'common/hooks/useModalState'
 import { getPlaylistLibrary } from 'common/store/account/selectors'
+import { make, useRecord } from 'common/store/analytics/actions'
```

Comment:
does it makes sense to have `useRecord` in actions? i guess it's a special action? and looks like that's what was there before anyway so nvm

Reply:
No it doesn't make sense, we can address in the other analytics issue I created